mbank-cli

```diff
@@ -1213,25 +1213,53 @@ sub do_login  ## no critic (ExcessComplexity)
         $error_code eq ''
             or scraping_error("login.2fa.init.error: $error_code");
         my $auth_mode = $data->{AuthMode} // '';
+        my $content = '';
+        my $tran_id = '';
         if ($auth_mode eq 'NAM') {
-            known_bug(72, '2FA via mobile app is not supported yet');
+            print "Waiting for confirm click in mobile app...\n";
```

Comment:
All diagonostics must go to stderr, not stdout.

Reply:
That's informational message not diagnostic. In my wrapper script I assume that if stderr is not empty some kind of problem has appeared, but wating for confirm click is not a problem!

Reply:
I can see your point; but this breaks machine-readabilty of stdout, which is more important than machine-readability of stderr, IMO.

> Enter sms password prompt is not diagnostic too.

SMS passwd prompt is actually printed on tty:
```
$ ./mbank-cli debug-sms-password 2019-09-16 1 >/dev/null 2>&1
SMS password (2019-09-16, #1):
```
I guess we could do the same for this message. Would that would for you? And what do we do when there's no tty available?